applications/sasquatch/charts/strimzi-kafka/README.md

```diff
@@ -60,6 +61,7 @@ A subchart to deploy Strimzi Kafka components for Sasquatch.
 | kafka.listeners.external.enabled | bool | `false` | Whether external listener is enabled |
 | kafka.listeners.plain.enabled | bool | `false` | Whether internal plaintext listener is enabled |
 | kafka.listeners.tls.enabled | bool | `false` | Whether internal TLS listener is enabled |
+| kafka.metadataVersion | string | `nil` | The KRaft metadata version used by the Kafka cluster. @default: null --  If the property is not set, it defaults to the metadata version that corresponds to the version property. |
```

Comment:
This looks a bit ood, I think I had an extra ":" in my previous comment, sorry about that. Please try the following:

```
 # -- The KRaft metadata version used by the Kafka cluster.
  # @default null --  If the property is not set, it defaults to the metadata version that corresponds to the version property.
  metadataVersion: null
  ```